src/simulator_adapter.py

```diff
@@ -228,6 +238,8 @@ def obstacle_odom_callback(msg, topic, data={}):
     accel_pub = rospy.Publisher('acceleration', Vector3Stamped, queue_size=0)
     battery_pub = rospy.Publisher('fc_battery', Float32, queue_size=0)
     status_pub = rospy.Publisher('fc_status', FlightControllerStatus, queue_size=0)
+    if publish_ground_truth_localization:
```

Comment:
These are in the same method. Does it make sense to have two if statements for this? I do recognize that the method is split into publisher and subscriber creation but maybe it would be better to create publishers and subscribers in pairs?

Might even be something for a different PR